# debug_flag: true
pixel_ratio: 0.1
estimate/max_pose_error: 0.3 # main change this
estimate/drone_width: 0.3
estimate/drone_height: 0.2
